savaslabs/savaslabs.github.io
# Issue #5021: Comments restyling + Gravatars as React component.

Fixes issue [#5021](https://pm.savaslabs.com/issues/5021)

## Summary of changes

1. Adds `react-gravatar` package
1. Displays user avatars using md5 email hash. Default gravatar anonymous person is used for guests without gravatars. See here https://en.gravatar.com/site/implement/images/. Using provided default for now, will add a custom icon down the road.
1. Adds updated markup and styling for comments.

## Notes

this PR depends on https://github.com/savaslabs/squabble/pull/23

## To test

- [x] run Squabble server locally while on the branch linked above
- [x] Verify comment styling
- [x] submit comment using several different email addresses including the once you know have gravatars established (e.g. yours and my gmail)
- [x] verify gravatars are rendering immediately after the comment is submitted.

NOTE: I moved the comment form back to the bottom for the lack of time to properly adjust the animation on comment submit. When the form is at the top, submitted comment is displayed at the bottom of the page, but the won't see it because the page is not scrolled to the bottom. We should tweak this behavior down the road.

### Pull request reviewer

As the reviewer, I have verified the following:
